phases/19-capstone-projects/56-iteration-scheduler/code/main.py

```
@@ -0,0 +1,353 @@
+                stop_reason = "max_experiments"
+                break
+            if deadline_hit():
+                stop_reason = "deadline"
+                break
+
+            dispatch_until_full()
+
+        for task in list(in_flight.keys()):
+            try:
+                result = await task
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)